devops/openshift/load-test-gateway-deploy.yml

```diff
@@ -0,0 +1,196 @@
+                  value: "${DB_SERVICE}"
+                - name: POSTGRES_USER
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${DB_USERNAME_KEY}
+                      name: ${DB_SECRET_NAME}
+                - name: POSTGRES_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${DB_PASSWORD_KEY}
+                      name: ${DB_SECRET_NAME}
+                - name: POSTGRES_DB
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${SIMS_DB_NAME}
+                      name: ${LOAD_TEST_GATEWAY_SECRET_NAME}
+                - name: DISABLE_ORM_CACHE
+                  value: ${DISABLE_ORM_CACHE}
+                - name: ZEEBE_ADDRESS
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${ZEEBE_ADDRESS_KEY}
+                      name: ${LOAD_TEST_GATEWAY_SECRET_NAME}
+                - name: ZEEBE_CLIENT_ID
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${ZEEBE_CLIENT_ID_KEY}
+                      name: ${LOAD_TEST_GATEWAY_SECRET_NAME}
+                - name: ZEEBE_CLIENT_SECRET
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${ZEEBE_CLIENT_SECRET_KEY}
+                      name: ${LOAD_TEST_GATEWAY_SECRET_NAME}
+                - name: ZEEBE_AUTHORIZATION_SERVER_URL
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${ZEEBE_AUTHORIZATION_SERVER_URL_KEY}
+                      name: ${LOAD_TEST_GATEWAY_SECRET_NAME}
+              image: ${IMAGE_REGISTRY}/${BUILD_NAMESPACE}/${IMAGE_STREAM_TAG}
+              imagePullPolicy: Always
+              name: ${NAME}
+              ports:
+                - containerPort: "${{PORT}}"
+              resources:
+                limits:
+                  cpu: ${CPU_LIMIT}
+                  memory: ${MEMORY_LIMIT}
+                requests:
+                  cpu: ${CPU_REQUEST}
+                  memory: ${MEMORY_REQUEST}
+  - apiVersion: v1
+    kind: Service
+    metadata:
+      name: ${NAME}
+    spec:
+      ports:
+        - name: http
+          port: 80
+          targetPort: ${{PORT}}
+      selector:
+        deploymentconfig: ${NAME}
+      type: ClusterIP
+  - apiVersion: route.openshift.io/v1
+    kind: Route
+    metadata:
+      name: ${NAME}
+    spec:
+      host: ${HOST_NAME}
+      path: ${PATH}
+      tls:
+        insecureEdgeTerminationPolicy: Redirect
+        termination: edge
+      to:
+        kind: Service
+        name: ${NAME}
+  - apiVersion: autoscaling/v1
+    kind: HorizontalPodAutoscaler
+    metadata:
+      name: ${NAME}-hpa
+    spec:
+      scaleTargetRef:
+        apiVersion: apps/v1
+        kind: ReplicationController
+        name: ${NAME}-rc
+      minReplicas: "${{REPLICAS}}"
+      maxReplicas: 10
+      targetCPUUtilizationPercentage: 80
```

Comment:
Do we need autoscaling for this one?

Reply:
I would recommend not using it. Before the test, it would be good to have them scaled to what we believe would be best to avoid new pods creation.